resources/lib/menu_functions.py

```diff
@@ -423,6 +423,25 @@ def getCollections(detailsString):
     collections.append(item_data)
 
     item_data = {}
+    item_data['title'] = i18n('tvshows_latest')
+    item_data['media_type'] = 'Episodes'
+    item_data['path'] = ('{server}/emby/Users/{userid}/Items/Latest' +
+                         '?Limit={ItemLimit}' +
+                         '&Recursive=true' +
+                         '&GroupItems=true' +
+                         '&SortBy=DateCreated' +
+                         '&Fields=' + detailsString +
+                         '&SortOrder=Descending' +
+                         '&Filters=IsUnplayed' +
+                         '&IsVirtualUnaired=false' +
+                         '&IsMissing=False' +
+                         '&IncludeItemTypes=Episode' +
+                         '&ImageTypeLimit=1' +
+                         '&format=json')
+    item_data['name_format'] = 'episode_name_format'
+    collections.append(item_data)
```

Comment:
Everything in Latest is in Unwatched but not the other way around.  Latest is limited to some data cutoff, or maybe item limit by the server.  If Unwatched can be sorted by when the entire series was added, but not be when it was most modified (from what I can tell).  Unwatched sorted by the latest episode add date would be the same - but from what I can tell you can't do that.